test/README.md

```diff
@@ -0,0 +1,16 @@
+# Test
+
+This directory contains tests and testing docs.
+
+* [Unit tests](#running-unit-tests) currently reside in the codebase alongside the code they test
+
+## Running unit tests
+
+TODO(#66): Write real unit tests.
```

Comment:
this is the wrong issue number for this repo.

Reply:
Replaced with proper instructions, now that we have unit tests in place.